src/twisted/logger/_filter.py

```diff
@@ -154,15 +151,13 @@ def logLevelForNamespace(self, namespace):
         C{logLevelForNamespace("mypackage.subpackage")} will return
         C{LogLevel.debug}.
 
-        @param namespace: A logging namespace, or L{None} for the default
+        @param namespace: A logging namespace.  Use C{""} for the default
             namespace.
-        @type namespace: L{str} (native string)
 
         @return: The log level for the specified namespace.
-        @rtype: L{LogLevel}
         """
         if not namespace:
-            return self._logLevelsByNamespace[None]
+            return self._logLevelsByNamespace[""]
```

Comment:
Notice here that we treat `None` and `""` as the same thing, and always return the same/shared log level for either namespace.